views/help.jade

```diff
@@ -5,27 +5,121 @@ block content
     h1 Help
     ul
         li
-            a(href='#one') What is CC0 and public domain?
+            a(href='/help/createslide') Create Slides
         li
-            a(href='#two') Why Does copyright matter?
+            a() Adding Songs/Translations to Glorify.cc
```

Comment:
we will eventually redirect it to login if the user not logged in and addsong when the user is logged in. however, we haven't implemented the feature where after log in it has the capability to go back to where it was